### DISCUSSION TITLE
[Bug] can't upload or link to PDF

### DISCUSSION THREAD

**Discussion by jasondeglint** [webstudio-is/webstudio-community#152] [Q&A] [answered]
According to this tutorial you can link to PDFs:
https://docs.webstudio.is/university/core-components/link#:~:text=attachments%20such%20as-,PDFs,-%2C%20documents%2C%20or%20media

However when I try to upload a PDF file I get the following error:
"Unknown font format"

Please help. Thanks! :)

**Comment by kof** [comment, marked as answer]
We don't support pdf uploads at the moment, the link would need to point somewhere else

**Reply by jasondeglint** [reply]
Amazing - thanks for clarifying that.

Is there a timeline on when PDF upload and linked will be supported?

**Reply by kof** [reply]
We haven't decided on this. For now use other services like dropbox

**Reply by kof** [reply]
You can certainly link to other files via url
